website/content/docs/connect/config-entries/exported-services.mdx

```diff
@@ -221,7 +221,7 @@ A asterisk wildcard (`*`) cannot be specified as the `Partition`.
 ## Examples
 
 
-### Exporting services to peered clusters
+### Exporting specific services
```

Comment:
A datacenter-local partition is not a "peered cluster", so I've updated this heading accordingly